app/models/rbac.rb

```diff
@@ -463,16 +461,32 @@ def self.search(options = {})
     ids_clause             = nil
     target_ids             = nil
 
-    unless targets.empty?
+    if targets.nil? || targets.kind_of?(Array) && targets.empty?
+      scope = apply_scope(klass, scope)
+    elsif targets.kind_of?(Array) && !targets.empty?
```

Comment:
I want empty arrays to get here.
hmm
or I guess we could just return `[]` if the array is empty